Software/Development/SemesterProject/System/Include/scheduler.h

```diff
@@ -48,6 +48,7 @@ void runTask(void (*functionPTR)());
 
 void task50us(void);
 void task100us(void);
+void task1ms(void);
```

Comment:
Maybe 1ms is too much, during tests, we were printing at 10ms and it was crashing (true, with the long names but still....)

Reply:
An idea is to call the task something like taskUART(void) and then set the cyclicity to a constant that we can change depending on the case...